manifests/components/fluentd-es-config/output.conf

```diff
@@ -6,23 +6,21 @@
 <match **>
   @id elasticsearch
   @type elasticsearch
-  @log_level info
-  type_name fluentd
+  @log_level error
   include_tag_key true
   host "#{ENV['ES_HOST']}"
   port 9200
   logstash_format true
-  <buffer>
-    @type file
-    path /var/log/fluentd-buffers/kubernetes.system.buffer
-    flush_mode interval
-    retry_type exponential_backoff
-    flush_thread_count 2
-    flush_interval 5s
-    retry_forever
-    retry_max_interval 30
-    chunk_limit_size 2M
-    queue_limit_length 8
-    overflow_action block
-  </buffer>
+  # Set the chunk limits.
+  buffer_chunk_limit 2M
+  buffer_queue_limit 8
+  # Bitnami: 5s -> 30s
+  flush_interval 30s
+  # Never wait longer than 5 minutes between retries.
+  max_retry_wait 30
+  # Bitnami: retry 5 times most instead of forever (our elasticsearch deploy is capped to 2G mem)
+  disable_retry_limit false
+  retry_limit 5
```

Comment:
Does this mean we try 5 times, and then lose the logs?
(that's ok, I just want to understand the implications and overall timeout behaviour)